static/app/views/dashboards/widgets/timeSeriesWidget/releaseBubbles/useReleaseBubbles.tsx

```diff
@@ -230,34 +230,24 @@ function ReleaseBubbleSeries({
       //  ----------------  ----------------
       //                 ^  ^
       //                 |--|
-      //                 bubblePadding.x
-
-      // No left-padding on first bubble
-      x: bubbleStartX + (params.dataIndex === 0 ? 0 : bubblePadding.x),
-
-      // No right-padding on last bubble
-      // We don't decrease width on first bubble to make up for lack of
-      // left-padding
-      width:
-        width -
-        (params.dataIndex === 0 || params.dataIndex === data.length - 1
-          ? 0
-          : bubblePadding.x),
```

Comment:
this needed to be `bubblePadding.x * 2` to account for the shift in `x` (left side) *and* one on the right side